docs/learn/learn-staking-advanced.md

```diff
@@ -1,42 +1,48 @@
 ---
 id: learn-staking-advanced
 title: Advanced staking
-sidebar_label: Advanced staking
+sidebar_label: Staking
```

Comment:
I see your reasoning for removing the word "Advanced", but it makes sense to keep it

```suggestion
sidebar_label: Advanced Staking Concepts
```